tm_vec/.ipynb_checkpoints/tm_vec_utils-checkpoint.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
```

Comment:
@tymor22 what's going on in this file? Why are there python files saved under .ipynb_checkpoints?